src/AirSpeedSensor.cc

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#if defined(_MSC_VER)
+  #pragma warning(push)
+  #pragma warning(disable: 4005)
+  #pragma warning(disable: 4251)
+#endif
+#include <gz/msgs/air_speed_sensor.pb.h>
+#if defined(_MSC_VER)
+  #pragma warning(pop)
+#endif
+#include <gz/msgs/Utility.hh>
+
+#include <gz/common/Profiler.hh>
+#include <gz/transport/Node.hh>
+
+#include "gz/sensors/GaussianNoiseModel.hh"
+#include "gz/sensors/Noise.hh"
+#include "gz/sensors/SensorTypes.hh"
+#include "gz/sensors/SensorFactory.hh"
+#include "gz/sensors/AirSpeedSensor.hh"
+
+using namespace gz;
+using namespace sensors;
+
+static constexpr double kPressureOneAtmospherePascals = 101325.0;
+// static constexpr double kSeaLevelTempKelvin = 288.15;
+
+static constexpr auto DEFAULT_HOME_ALT_AMSL = 0.0f; // altitude AMSL see level [m]
+
+// international standard atmosphere (troposphere model - valid up to 11km) see [1]
+static constexpr auto TEMPERATURE_MSL = 288.15f; // temperature at MSL [K] (15 [C])
+static constexpr auto PRESSURE_MSL = 101325.0f; // pressure at MSL [Pa]
+static constexpr auto LAPSE_RATE = 0.0065f; // reduction in temperature with altitude for troposphere [K/m]
+static constexpr auto AIR_DENSITY_MSL = 1.225f; // air density at MSL [kg/m^3]
```

Comment:
nit: this follows the style guide

```suggestion
static constexpr auto kDefaultHomeAltAmsl = 0.0f; // altitude AMSL sea level [m]

// international standard atmosphere (troposphere model - valid up to 11km) see [1]
static constexpr auto kTemperaturMsl = 288.15f; // temperature at MSL [K] (15 [C])
static constexpr auto kPressureMsl = 101325.0f; // pressure at MSL [Pa]
static constexpr auto kLapseRate = 0.0065f; // reduction in temperature with altitude for troposphere [K/m]
static constexpr auto kAirDensityMsl = 1.225f; // air density at MSL [kg/m^3]
```